gffquant/profilers/feature_quantifier.py

```
@@ -55,26 +56,52 @@ def __init__(
+        if external_genecounts:
+            self._import_counts(external_genecounts)
+
+    def _import_counts(self, fn):
+        with get_open_function(fn)(fn, "rt") as _in:
+            try:
+                _ = next(_in)
+            except StopIteration as exc:
+                raise ValueError(f"Counts file is empty: {fn}") from exc
+
+            gene_id, *counts = next(_in).strip().split("\t")
+            ncols = len(counts)
```

Flake8 (7.3.0)

[error] local variable 'ncols' is assigned to but never used

(F841)